wrangler-core/src/main/java/io/cdap/wrangler/schema/Constants.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.wrangler.schema;
+
+/**
+ * Constants for schema handling in Wrangler
+ */
+public class Constants {
+  /**
+   * Constants for storing Workspace schema in TransientStore
+   */
+  public static final class TransientStoreKeys {
```

Comment:
Do you really need 2 levels? We already have a number of `Constants` classes in the project and it makes it hard to select which one with completion when writing code